src/03_modules.py

```diff
@@ -5,27 +5,36 @@
 level operating system functionality.
 """
 
+import platform
+import os
 import sys
 # See docs for the sys module: https://docs.python.org/3.7/library/sys.html
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
+print(str(sys.argv))
```

Comment:
@RodBromberg: Good work! Don't forget that you may want to use a 'for' loop here in case `sys.argv` has more than one argument! 👍